tools/db_crashtest.py

```diff
@@ -121,14 +121,32 @@
 }
 
 _TEST_DIR_ENV_VAR = 'TEST_TMPDIR'
+_DISK_TEMP_DIR_ENV_VAR = "DISK_TEMP_DIR"
+_HOME_ENV_VAR = "HOME"
+
+
+def need_direct_io_support(params):
+    return (params["use_direct_reads"] == 1 or \
```

Comment:
What if these parameter values are randomly chosen?

Reply:
no matter how they are chosen, if the params satisfy these conditions, then direct IO support is needed. What's your concern on the random chosen params?